torchx/runner/api.py

```diff
@@ -324,6 +324,10 @@ def dryrun(
 
         configured_trackers = get_configured_trackers()
 
+        app_handle = make_app_handle(scheduler, self._name, macros.app_id)
+
+        os.environ[ENV_TORCHX_JOB_ID] = app_handle
```

Comment:
what is the purpose of setting the `app_handle` in the current process' environment variables?

Reply:
Hi there, i think i exported the wrong version of my changes to a PR. Let me revise this and update.